install-deps.sh

```diff
@@ -12,6 +12,14 @@
 #  version 2.1 of the License, or (at your option) any later version.
 #
 set -e
+
+if ! [ "${_SOURCED_LIB_BUILD}" = 1 ]; then
+    SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+    CEPH_ROOT="${SCRIPT_DIR}"
+    . "${CEPH_ROOT}/src/script/lib-build.sh" || exit 2
```

Comment:
I'll think about it. Since these are all *bash* scripts, I don't think we should hesitate to use bash specific features. Personally, I think some of the code could be improved by use of bash arrays, for example.